Gemfile

```diff
@@ -131,6 +133,8 @@ source "https://rails-assets.org" do
   gem "rails-assets-angular-mocks",                   "~>1.5.0"
   gem "rails-assets-angular-patternfly-sass",         "~>3.3.2"
   gem "rails-assets-angular-sanitize",                "~>1.5.0"
+  gem 'rails-assets-ui-router',                       "~>0.2.18"
+  gem "rails-assets-bootstrap-datepicker",            "~>1.4.0"
```

Comment:
I think I had problems without it, so I put it in gem file. But I will remove both o these before finishing this PR, since we will use bower-rails.